package.json

```diff
@@ -14,47 +14,18 @@
   "name": "@polymer/iron-menu-behavior",
   "license": "BSD-3-Clause",
   "devDependencies": {
-    "@polymer/gen-typescript-declarations": "^1.2.2",
-    "bower": "^1.8.0",
     "webmat": "^0.2.0",
     "@polymer/paper-styles": "^3.0.0-pre.20",
-    "@polymer/iron-component-page": "^3.0.0-pre.20",
     "@polymer/iron-test-helpers": "^3.0.0-pre.20",
     "wct-browser-legacy": "^1.0.1",
     "@webcomponents/webcomponentsjs": "^2.0.0"
   },
   "scripts": {
-    "update-types": "bower install && gen-typescript-declarations --deleteExisting --outDir .",
-    "format": "webmat && npm run update-types"
+    "format": "webmat"
   },
   "version": "3.0.0-pre.20",
-  "resolutions": {
-    "inherits": "2.0.3",
-    "samsam": "1.1.3",
-    "supports-color": "3.1.2",
-    "type-detect": "1.0.0",
-    "@webcomponents/webcomponentsjs": "2.0.0-beta.2"
-  },
   "contributors": [
-    "T",
```

Comment:
😮

Reply:
Not that it matters, but I've been moving this to "author" not "contributors"